src/library/browse/browsethread.cpp

```diff
@@ -211,7 +211,7 @@ void BrowseThread::populateModel() {
             QString duration = pTrack->getDurationText(mixxx::Duration::Precision::SECONDS);
             item = new QStandardItem(duration);
             item->setToolTip(item->text());
-            item->setData(item->text(), Qt::UserRole);
+            item->setData(pTrack->getDurationInt(), Qt::UserRole);
```

Comment:
Does `double getDuration()` also work? No need to round to integers here.

Reply:
Oh, of course, yeah that also works.